lib/octokit/client/source_import.rb

```diff
@@ -88,6 +109,35 @@ def cancel_source_import(repo, options = {})
         options = ensure_api_media_type(:source_imports, options)
         boolean_from_response :delete, "#{Repository.path repo}/import", options
       end
+
+      # List source import large files
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository.
+      # @param options [Hash]
+      # @option options [Fixnum] :page Page of paginated results
```

Comment:
Is this option something other than normal GitHub pagination?

If it isn't we should ✂️  this line since we don't talk about pagination in any of the other methods.